### PR TITLE
88 - Onboarding Components

### PR DESCRIPTION
**Pull request by BenJamminnn**
# Onboarding Components

Credit to @ryansheh for the work, this PR adds those files while resolving complex pbxproj conflicts

### CODE REVIEW

**Comment by ryansheh** [comment]
I think there are a few things missing from my branch:
1) I deleted AuthManager and Keychain and consolidated all that logic into AccountKeychain.
2) I also made changes in User, TabBarViewModel  and SettingsVc

**Comment by ryansheh** [comment]
Could you please also update the production flow to launch the onboarding screen, and nav the user home when finished?
The logic begins at SceneDelegate "#if DEBUG"

File: DSNP-Wallet/Source/Modules/GenerateKeysViewController.swift [hunk 1]
```diff
@@ -18,11 +18,13 @@ class GenerateKeysViewController: UIViewController {
     }
     
     private func manageKeys() {
-        if let keys = AuthManager.shared.loadKeys() {
+        
+        do {
+            let keys = try DSNPWallet().loadKeys()
```
Comment (BenJamminnn) [inline DSNP-Wallet/Source/Modules/GenerateKeysViewController.swift]: @ryansheh Should we be using `loadKeys()` here or the `fetchKey()` on the `AccountManager`? I notice this func wants DSNP keys but the other function returns `Data`

Comment (ryansheh) [reply DSNP-Wallet/Source/Modules/GenerateKeysViewController.swift]: This class is actually safe to remove. It was used as a placeholder vc for testing until this  onboarding epic was made.

**Review by ryansheh — APPROVED** [review]

**Pull request closed without merging**